get_contributors.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Get a list of all contributors that contributed in some way
+# either by directly opening a PR, or by participating in issues.
+#
+# This script is a helper to fill out the "Contributors" chapter.
+# It'll only spit out the names and an user must add them to the book.
+#
+# This script takes two arguments:
+#   - `username`: Your github username used to authenticate the GitHub API
+#   - `token`: A github Personal Access Token used to authenticate the GitHub API
+
+set -euo pipefail
+
+# Check if there are `username` and `token` arguments
+if [ $# -eq 0 ]
+then
+  echo "Usage: $0 <username> <token>"
+  exit 1
+fi
+
+user="$1"
+token="$2"
+
+# Check if a command is available, otherwise exit.
+function check_bin() {
+  if ! command -v $1 &> /dev/null
+  then
+    echo "'$1' is not installed, but required to run this script."
+    exit 1
+  fi
+}
+
+check_bin curl
+check_bin jq
+
+# Get a list of users that participated in issues.
+function issue_contributors() {
+  local numbers=$(curl -s -u $user:$token -H "Accept: application/vnd.github.v3+json" 'https://api.github.com/repos/rust-lang/wg-async-foundations/issues?state=all&labels=status-quo-story-ideas' | jq '.[].number')
```

Comment:
Hmm, I was reading over this and I got to thinking -- this doesn't include commenting on PRs, does it?

Reply:
Do we want to include that?

Reply:
Also, if so, should we just include all PRs or do we want to label those that are contributing stories...

Reply:
It does include all pull requests with the `status-quo-story` label. GitHubs REST API v3 treats every pull request as an issue.

Reply:
OK, so if we just go and add those labels, we should be all set...


...we should extend it with shiny-future labels too I guess

Reply:
Yes that makes sense. Should I open a new PR that fixes it?

Reply:
Yep, thanks.

Reply:
@nikomatsakis I just noticed that there is no `shiny-story` label yet. You'ld have to create one

Reply:
Done